enabler/src/components/payment-methods/card/card.ts

```diff
@@ -1,13 +1,25 @@
-import { ComponentOptions, PaymentMethod } from '../../../payment-enabler/payment-enabler';
+import { ComponentOptions, PaymentComponent, PaymentComponentBuilder, PaymentMethod } from '../../../payment-enabler/payment-enabler';
 import buttonStyles from '../../../style/button.module.scss';
 import inputFieldStyles from '../../../style/inputField.module.scss';
 import styles from '../../../style/style.module.scss';
 import { BaseComponent, BaseOptions } from '../../base';
 import { addFormFieldsEventListeners, fieldIds, getCardBrand, getInput, validateAllFields } from './utils';
 
+export class CardBuilder implements PaymentComponentBuilder {
```

Comment:
Lets specify that this component supports the submit... Other option would be to do it on the `base.ts` file to make sure all components support the submit operation.

```suggestion
export class CardBuilder implements PaymentComponentBuilder {
  public componentHasSubmit = true;
```

Reply:
Thanks @ddeliziact. Fixed it.